webapp/store/views.py

```diff
@@ -231,6 +235,30 @@ def brand_search_snap():
             status_code,
         )
 
+    def _get_file(file):
+        try:
+            with open(
+                os.path.join(flask.current_app.root_path, file), "r"
```

Comment:
We could maybe define the publishers yaml directory in settings and just append the publisher name to it?